virttest/ceph.py

```diff
@@ -240,6 +240,27 @@ def create_config_file(ceph_monitor):
     return ceph_cfg
 
 
+@error_context.context_aware
+def create_keyring_file(client_name, client_key):
```

Comment:
In current ceph related cases, we create keyring_file in the tmp folder of tp-libvirt code, and specify with --keyring when using it. Do we really need override ceph keyring in /etc/ceph folder?

Reply:
@chunfuwen ,
Yes, we did add --keyring for rbd command, for example below:
`rbd -m libvirtceph.usersys.redhat.com --keyring /tmp/avocado_iknmvl8y/ceph.key info ovmf-pool/bios.qcow2 && rbd -m libvirtceph.usersys.redhat.com --keyring /tmp/avocado_iknmvl8y/ceph.key rm ovmf-pool/bios.qcow2`
But after that, we also has qemu-img command, for example below, if no keyring file, it will return 'Operation not permitted' error when I use another ceph server.
`qemu-img convert -O raw /tmp/avocado_iknmvl8y/released_os.qcow2 rbd:ovmf-pool/bios.qcow2:mon_host=<mon_ip>`

Reply:
Command 'rbd -m libvirtauthceph.usersys.redhat.com --keyring /var/lib/avocado/data/avocado-vt/ceph.key info virtual-disk-ceph-pool/attach.img 2> /dev/null|| qemu-img convert -O raw /var/lib/avocado/data/avocado-vt/avocado-vt-vm1_test.img rbd:virtual-disk-ceph-pool/attach.img:mon_host=libvirtauthceph.usersys.redhat.com:id=admin:key=AQA8Ij5f97SyNxAAePrE+BysFp4M3KCD1JZB4Q==' finished with 0 after 0.4634408950805664s

it is one of output from running ceph case, does it satisfy your request?